src/figuresClasses.ts

```diff
@@ -1,19 +1,64 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue' | string;
```

Comment:
make types or enum for shape and color and use it everywhere.
for example.
```
enum Shape {
   tirangle = 'triangle',
   circle = 'circle',
   rectangle = 'rectangle',
}
```

Reply:
Really good solution - I couldn't figure out how to write it correctly before!🙄 Thanks!!!🤝🔥